cmd/observe/flows.go

```diff
@@ -345,13 +345,13 @@ more.`,
 
 	filterFlags.Var(filterVar(
 		"from-service", ofilter,
-		"Show all flows originating in the given service ([namespace/]<svc-name>). If namespace is not provided, 'default' is used"))
+		"Show flows where the destination IP address matches the ClusterIP address originating in the given service ([namespace/]<svc-name>). If namespace is not provided, 'default' is used"))
 	filterFlags.Var(filterVar(
 		"service", ofilter,
-		"Show all flows related to the given service ([namespace/]<svc-name>). If namespace is not provided, 'default' is used"))
+		"Show flows where the destination IP address matches the ClusterIP address related to the given service ([namespace/]<svc-name>). If namespace is not provided, 'default' is used"))
```

Comment:
`--service` will match either the source or destination IP, not only the destination IP
```suggestion
		"Shows flows where either the source or destination IP address matches the ClusterIP address of the given service ([namespace/]<svc-name>). If namespace is not provided, 'default' is used"))
```